CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 #### Upcoming Changes
 
+* Fix CI: [#1825](https://github.com/lambdaclass/cairo-vm/pull/1825):
```

Comment:
Since this is just a CI fix we don't included in the changelog
Add the CI label to the PR to pass the Changelog check